tests/test_artifacts.py

```diff
@@ -43,8 +43,8 @@
 }
 FOLDER_INFO = ArtifactFolderInfoResponse(**FOLDER_INFO_RESPONSE)
 FILE_INFO_RESPONSE = {
-    "repo": ARTIFACT_FOLDER,
-    "path": ARTIFACT_PATH,
+    "repo": ARTIFACT_REPO,
+    "path": ARTIFACT_SHORT_PATH,
```

Comment:
The above line actually fixes a minor bug, as previously the `path` would include the repo, which is wrong.